packages/iov-ethereum/src/testconfig.ts

```diff
@@ -35,6 +39,10 @@ const local: EthereumNetworkConfig = {
   nonce: new Int53(0) as Nonce,
   gasPrice: "20000000000",
   gasLimit: "2100000",
+  waitForTx: 100,
+  scraperApi: "https://api-ropsten.etherscan.io/api",
```

Comment:
Does api-ropsten.etherscan.io index your local chain?

You can make this optional:

```ts
  readonly scraper?: {
    readonly api: string;
    readonly chainId: ChainId;
    readonly address: Address;
  }
```

and use `scraper!.api` for chains you know `scraper` is set